std/conv.d

```diff
@@ -5593,7 +5594,8 @@ The result is true if and only if the input string is composed of whitespace
 characters (\f\n\r\t\v lineSep paraSep nelSep) and
 an even number of hexadecimal digits (regardless of the case).
 */
-private bool isHexLiteral(String)(in String hexData)
+@safe pure @nogc
```

Comment:
I know that templates infer that, but I wanted to guarantee it, because the types to it are constrained such that it can be guaranteed.